### DISCUSSION TITLE
Above 4G Decode alread enabled?

### DISCUSSION THREAD

**Discussion by Raboo** [xCuri0/ReBarUEFI#194] [Q&A] [answered]
Hi,

I have a ASRock Z97 Extreme6, Haswell CPU.
Can i trust GPU-Z when it says `Above 4G Decode enabled in BIOS = Yes` or should I still follow the guide in [Enabling-hidden-4G-decoding](https://github.com/xCuri0/ReBarUEFI/wiki/Enabling-hidden-4G-decoding)?
Please note that I don't have an option to control 4G Decode in UEFI settings.

**Comment by xCuri0** [comment, marked as answer]
No, GPU-Z is incorrect often just follow the enabling hidden method

**Reply by Raboo** [reply]
ok, thanks
